projects/igniteui-angular/src/lib/test-utils/grid-cell-style-testing.scss

```diff
@@ -0,0 +1,17 @@
+:host::ng-deep {
+    .test2 {
+        color: black !important;
```

Comment:
Do we need `!important`?

Reply:
Yes, when a cell fits two or more conditions, we may want to define which would take precedence...not of great importance, but a possible use case.